spec/atlas/atlas_connectivity_spec.rb

```diff
@@ -14,9 +14,9 @@
   end
 
   describe 'connection to Atlas' do
-    it 'runs ismaster successfully' do
-      result = client.database.command(:ismaster => 1)
-      expect(result.documents.first['ismaster']).to be true
+    it 'runs hello successfully' do
+      result = client.database.command(:hello => 1)
+      expect(result.documents.first['isWritablePrimary']).to be true
```

Comment:
Did you test this manually? Atlas tests aren't run as part of the regular suite in evergreen.

Reply:
You're right, I forgot we do not run atlas specs on evergreen. I reverted the changes since I cannot test them manually. Let's do it in a follow up PR.

Reply:
:+1: